nmostesting/mocks/Node.py

```diff
@@ -68,7 +68,7 @@ def get_sender(self, stream_type="video"):
         }
         return sender
 
-    def add_sender(self, sender, sender_ip_address):
+    def add_sender(self, sender, sender_ip_address, sdp_params):
```

Comment:
This function is also used in `IS0404Test.test_05` so does that need fixing up, or do you want a default argument for `sdp_params` here?

Reply:
Yes, good point - default for `sdp_params` I think